sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/parquet/ParquetFilters.scala

```diff
@@ -238,6 +238,14 @@ private[parquet] object ParquetFilters {
       case sources.Not(pred) =>
         createFilter(schema, pred).map(FilterApi.not)
 
+      case sources.In(name, values) if canMakeFilterOn(name) =>
```

Comment:
Always push-down? Should we also consider the number of elements in `values`? What is the performance impact when the number of values is around 10 or more?